Engines/Wine/QuickScript/Zip Script/script.js

```diff
@@ -57,7 +57,8 @@ ZipScript.prototype.go = function () {
         .wizard(setupWizard)
         .archive(archive)
         .to(wine.prefixDirectory() + "/drive_c/" + this._name)
-        .extract();
+        .extract()
+        .wine.run();
```

Comment:
You must pass the extracted executable to `wine.run()`. See e.g. https://github.com/PhoenicisOrg/scripts/blob/749c609a32aace5376853af643b7d8ebbe012921/Engines/Wine/QuickScript/Steam%20Script/script.js#L129

Reply:
So it's runInsidePrefix instead of just run?

Reply:
Yes.